Sources/StreamChat/Repositories/SyncRepository_Tests.swift

```diff
@@ -190,10 +196,11 @@ final class SyncRepository_Tests: XCTestCase {
 
         if let result = requestResult {
             // Simulate API Failure
-            AssertAsync {
-                Assert.willNotBeNil(self.apiClient.request_completion)
+            AssertAsync.willBeTrue(apiClient.recoveryRequest_completion != nil)
+            guard let callback = apiClient.recoveryRequest_completion as? (Result<MissingEventsPayload, Error>) -> Void else {
+                XCTFail()
```

Comment:
Can we populate the `XCTFail` call here? If this test fails on CI it will be hard to reason about the issue.